dotCMS/src/main/resources/es-content-mapping.json

```diff
@@ -47,11 +65,28 @@
             	"tags" : {
 		            "match": "tags",
 		            "mapping": {
-		                "type": "string",
+		                "type": "keyword",
 		                "analyzer": "dot_comma_analyzer",
 		                "doc_values": false 	
 					}
 	            }
+            },
+            {
+                "textmapping": {
+                    "match_mapping_type": "string",
+                    "mapping": {
+                        "analyzer": "whitespace",
+                        "search_analyzer": "whitespace",
+                        "store": "false",
+                        "type": "text",
+                        "fields": {
+                            "_keyword": {
```

Comment:
do we need the "keyboard" field even having the "dotraw"? available?

Reply:
I have the same doubt, I'd like to ask @wezell before removing _keyword fields. So far, _keyword fields are used for aggregations because text fields do not support it. I'd rather prefer to use _dotraw because they have the same value and datatype.

Reply:
@nollymar can you please add this to a google doc or notes of things to ask @wezell when he comes back

Reply:
sure!